impl/src/test/java/org/ehcache/internal/store/heap/BaseOnHeapStoreTest.java

```diff
@@ -618,7 +679,7 @@ public String apply(String mappedKey) {
           throw RUNTIME_EXCEPTION;
         }
       });
-      throw new AssertionError();
+      fail("Expected exception");
```

Comment:
To add a message in case this ever fails, applying boy scout rule while navigating (and changing this code)